frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -84,8 +86,15 @@ class TripsContainer extends React.Component {
 
   /** @override */
   render() {
+    if (this.state.trips === undefined || this.state.trips.length === 0) {
```

Comment:
What happens if an accordion with no trips is returned? Is it different from the empty div tags?

Reply:
Hmm I thought that I checked this originally and it had an error but it must not have because it seems to work the same as empty div tags. Removed the conditional block.